content/blog/2020/06/2020-06-12-infra-and-aws-donation.adoc

```diff
@@ -0,0 +1,83 @@
+
+During April 2020:
+
+* Over 600 000 people visited link:https://www.jenkins.io/[www.jenkins.io]
+* Over 250 000 Jenkins servers regularly checked the
+  link:https://pkg.jenkins.io/[Jenkins package server] and the
+  link:https://updates.jenkins.io/[Jenkins update server]
+* Over 43 000 continuous integration jobs ran on link:https://ci.jenkins.io/[ci.jenkins.io]
+* Over 950 plugins ran their continuous integration pipelines on link:https://ci.jenkins.io/[ci.jenkins.io]
+
+All those services require infrastructure.
+The Jenkins project relies heavily on its infrastructure.
+We use websites like link:/[www.jenkins.io] and link:https://plugins.jenkins.io/[plugins.jenkins.io], ticketing systems like link:https://issues.jenkins.io/[issues.jenkins.io]  , CI/CD infrastructure like link:https://ci.jenkins.io/[ci.jenkins.io] , and many other services.
+And like any organization, there are specific people who make sure that those services are always up and running.
+
+The Jenkins project, as an open source project, is built and maintained by its awesome community of users.
+Everyone is welcome to participate.
+Infrastructure is no exception.
+
+We have a vision to be a community effort trying to be as transparent as possible.
+What better way than using git to manage infrastructure work?
+
+== Who said GitOps?
+
+Since the creation of the link:https://github.com/jenkins-infra[Jenkins-infra organization] on GitHub in March 2008, more than 650 people have contributed to over 80 git repositories.
+Those contributions make the Jenkins community what it is today.
+If you can't find something there, it probably means that some help is welcomed.
+
+More recently, with help from link:https://github.com/halkeye[Gavin Mogan], link:https://github.com/timja[Tim Jacomb], and link:https://github.com/slide[Alex Earl], big achievements have been possible on many fronts like automating jenkins releases, refreshing link:https://plugins.jenkins.io/[plugins.jenkins.io], adding new agents to link:https://ci.jenkins.io/computer/[ci.jenkins.io], and maintaining our kubernetes cluster.
+We thank them for their help and for the infrastructure progress they have enabled.
+
+== Infrastructure at Scale
+
+Running infrastructure at the scale the Jenkins project does is expensive and sometimes quite challenging.
+We are fortunate enough to be supported by many leading companies that provide us their expertise, their products, and their support.
+
+Recently, Amazon Web Services donated $60 000 to run Jenkins infrastructure on the AWS cloud.
+We're so grateful for their donation and for the flexibility it provides.
+We're running Linux agents with AMD64 and ARM64 architectures on AWS.
+We're using AWS cloud for our Windows agents.
+The generous infrastructure donation from Amazon Web Services has increased our continuous integration capacity and broadened our platform coverage.
+
+== Our Sponsors
+
+Sponsors of the Jenkins project include
```

Comment:
Add an image with all logos? 
It might also make sense to sort the sponsors in alphabetical order, and maybe split into two lists (major contributors and minor ones, similar to https://www.jenkins.io/).

Reply:
Sorted into the order that is used on www.jenkins.io for the major sponsors and alphabetized the other sponsors in a second sentence.  Hyperlinked to all of them.

Reply:
Think that is nice to have but maybe save for another post so we can ship this one sooner?